worker/remoterelations/remoteapplicationworker.go

```diff
@@ -120,7 +120,7 @@ func (w *remoteApplicationWorker) remoteOfferRemoved() error {
 func (w *remoteApplicationWorker) loop() (err error) {
 	// Watch for changes to any remote relations to this application.
 	relationsWatcher, err := w.localModelFacade.WatchRemoteApplicationRelations(w.applicationName)
-	if errors.IsNotFound(err) {
+	if err != nil && params.IsCodeNotFound(err) {
```

Comment:
Not hydrating errors os a real problem for us here. Knowing when to use `params` vs `errors`.